docs/source/release/v3.14.0/framework.rst

```diff
@@ -90,6 +90,7 @@ Improvements
 ############
 
 - :ref:`ChudleyElliot <func-ChudleyElliot>` includes hbar in the definition
+- :ref:`Functions <FitFunctionsInPython>` may now have their constraint penalties for fitting set in python using ``function.setConstraintPenaltyFactor(double)``.
```

Comment:
Function signature is `(string, double)`

Reply:
This has been corrected.